platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/tasks_sample.adoc

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 2023 Gradle, Inc.
+//
+// Licensed under the Creative Commons Attribution-Noncommercial-ShareAlike 4.0 International License.;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://creativecommons.org/licenses/by-nc-sa/4.0/
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[structure_large_projects]]
+= Tasks Sample
+
+This sample shows how to create a plugin with tasks that have dependency resolution results as inputs.
```

Comment:
❓ What is the purpose of including this sample at this point?  This uses lots of concepts that haven't been explained yet.  The dependency resolution parts in particular will be impenetrable to newcomers, and the Plugin itself is more complex than "just adds a bunch of tasks".

Reply:
Samples have been removed.